includes/Ajax.php

```diff
@@ -1033,4 +1046,32 @@ public function dismiss_pro_notice() {
 
         wp_send_json_success();
     }
+
+    /**
+     * Handler for reCaptcha validation request.
+     *
+     * @since 3.3.2
+     *
+     * @param string $token
+     * @param string $sitekey
+     * @param string $action
+     *
+     * @return boolean
+     */
+    public function recaptcha_validation_handler( $sitekey, $token, $action ) {
+        if ( empty( $sitekey ) || empty( $token ) ) {
+            return false;
+        }
+
+        $siteverify = 'https://www.google.com/recaptcha/api/siteverify';
+        $response   = file_get_contents( $siteverify . '?secret=' . $sitekey . '&response=' . $token );
+        $response   = json_decode( $response, true );
+
+        // Validate reCaptcha action
+        if ( $action !== $response['action'] ) {
+            return false;
+        }
+
+        return $response['success'];
```

Comment:
If`success` key not exist in response then there will be fatal error.

Reply:
Bhai, reCaptcha API always returns `$response['success']`. Either `true` if the request is valid or `false` if there is something wrong.